master-firmware/src/strategy.cpp

```diff
@@ -69,6 +75,11 @@ bool strategy_goto_avoid(struct _robot* robot, int x_mm, int y_mm, int a_deg)
     point_t *points;
     int num_points = oa_get_path(&points);
     DEBUG("Path to (%d, %d) computed with %d points", x_mm, y_mm, num_points);
+    if (num_points <= 0) {
+        WARNING("No path found!");
```

Comment:
Good enough for now, but I think we might want to start a recovery strategy here instead.